.travis.yml

```diff
@@ -308,6 +308,7 @@ script:
             rm -rf curl-99.98.97
             # verify out-of-tree build
             (tar xf curl-99.98.97.tar.gz && \
+             touch docs/{cmdline-opts,libcurl}/Makefile.inc && \
```

Comment:
this touches in the wrong dir, it needs to be prefixed with `curl-99.98.97`. But I don't think this triggers an error if there's one or two .d files missing in the cmdline-opts directory, does it?